packages/big-design/src/components/Select/Select.tsx

```diff
@@ -120,7 +131,7 @@ export class Select<T extends any> extends React.PureComponent<SelectProps<T>, S
   }
 
   private renderInput() {
-    const { placeholder, error, required, disabled, onChange, options, value } = this.props;
+    const { placeholder, error, filterable = true, required, disabled, onChange, options, value } = this.props;
```

Comment:
It should be fine. In this case since the prop is at the class level, it might be better to add it onto `defaultProps`. That way undefined doesn't get passed through the component.